docs/geedocs/docsrc/answer/geeServerAdmin/publishDefaultDatabase.rst

```diff
@@ -0,0 +1,38 @@
+|Google logo|
+
+========================
+Publish default database
+========================
+
+.. container::
+
+   .. container:: content
+
+      Google Earth Enterprise (GEE) Server allows users to publish one 3D
+      database as the default database. This is the database that will be
+      served when Earth Client signs into the base URL without including
+      a database name.
+
+      .. rubric:: Publishing as Default
+
+      To make a 3D database the default database, simply check the "Default
+      Database" checkbox when publishing the database.
+
+      |Publish as default|
+
+      This can also be done from the command line by including the ``--setecdefault``
+      option with the ``geserveradmin --publishdb`` command.
+      
+      e.g., ``geserveradmin --publishdb SFDb_3d --targetpath SFDb_3d --setecdefault``
+
+      .. rubric:: Viewing the Default Database
+
+      To view the default database in Earth Client, simply sign into the base
```

Comment:
Minor: this is getting picky, but I wonder if we should include a note indicating that if you publish a database as the default, any previous default will be cleared.

Reply:
Added a clarification. Check the wording to see if it's clear.